File: app/src/main/java/com/example/fellow_traveller/Models/Car.java
package com.example.fellow_traveller.Models;

public class Car {
    private int id;
    private String brand;
    private String model;
    private String plate;

    public Car(String brand, String model, String plate) {
        this.brand = brand;
        this.model = model;
        this.plate = plate;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public String getPlate() {
        return plate;
    }

    public void setPlate(String plate) {
        this.plate = plate;
    }


    public String getDescription() {
       return brand + "  " + model + " | " + plate;
    }
}

File: app/src/main/java/gr/fellow/fellow_traveller/domain/Car.kt
package gr.fellow.fellow_traveller.domain

data class Car(
    val id: Int,
    val brand: String,
    val model: String,
    val plate: String,
    val color: String
) {
    val baseInfo: String
        get() = "$brand $model"

    val fullInfo: String
        get() = "$brand $model | $plate | $color"

}